blocktopmenu.php

```diff
@@ -1358,4 +1371,104 @@ public function renderList()
 
         return $helper->generateList($links, $fields_list);
     }
+
+    private function createCategoryNode(array $category, $addThumbnails = false)
+    {
+        if (!$category['active']) {
```

Comment:
`createCategoryNode` is the equivalent of `generateCategoriesMenu`, it does the same thing except it returns a node instead of HTML code. 
Moreover, since `blocktopmenu` is a front office module, I don't see the point of displaying inactive categories.